test/e2e/protocol/proposer.test.mjs

```diff
@@ -256,10 +257,12 @@ describe('Basic Proposer tests', () => {
 
         const res = await secondProposer.changeCurrentProposer();
         expectTransaction(res);
+        numChanges++;
       } catch (err) {
         console.log(err);
       }
     }
+    expect(numChanges).to.be.equal(8);
```

Comment:
You could extract the 8 into a const eg `CHAGE_PROPOSER_NO_TIMES`, then also update the condition in the for loop.